### PR TITLE
Update appdata.xml for duplicate events fix

### PR DESCRIPTION
**Pull request by mcclurgm**
Forgot to do this in #597 (sorry)

Is this the right way to start adding release notes after a release was just published?

### CODE REVIEW

**Review by jeremypw — DISMISSED** [review]
Yes, that's the way I would do it anyway. The correct version and date can be entered at time of actual release.

File: data/io.elementary.calendar.appdata.xml.in [hunk 1]
```diff
@@ -13,6 +13,13 @@
     </p>
   </description>
   <releases>
+    <release version="5.1.1" date="2020-09-02" urgency="medium">
+      <description>
+      <p>Minor Updates:</p>
+      <ul>
+        <li>Fix an issue where new events could appear twice in Calendar</li>
+      </description>
```
Comment (jeremypw) [inline data/io.elementary.calendar.appdata.xml.in:21]: Missing EndTag `<\ul>`

**Review by jeremypw — CHANGES_REQUESTED** [review]
There is a missing end tag, but other than that it looks fine.

File: data/io.elementary.calendar.appdata.xml.in (hunk 1 above)
Comment (mcclurgm) [reply data/io.elementary.calendar.appdata.xml.in:22]: Whoops, fixed it.

**Review by jeremypw — APPROVED** [review]
Good to go!

**Pull request merged**